doc/tutorials/plutus-tx.rst

```diff
@@ -102,33 +90,28 @@ Writing basic PlutusTx programs
    :start-after: BLOCK1
    :end-before: BLOCK2
 
-Here’s the most basic program we can write: one that just evaluates to
-the integer ``1``.
+This simple program just evaluates to the integer ``1``.
 
-.. note::
+.. **note::
 
-   We’ve included doctests to the examples that show the Plutus Core
-   that is generated from compilation. The syntax of Plutus Core will
-   look unfamiliar. This is fine, since it is the "assembly language"
-   and you won’t need to inspect the output of the compiler. However,
-   for our purposes it’s instructive to look at it to get a vague idea
-   of what’s going on.
+The examples that show the Plutus Core generated from compilation include doctests. The syntax of Plutus Core
```

Comment:
This was indented so it goes in the admonition. Or did you mean to move this into the body text?